### ISSUE TITLE
SOCKS access for Daniel Miller

### ISSUE THREAD

**Issue by DanielMiller-agile6** [department-of-veterans-affairs/va.gov-team#71627]
### Your Name

Daniel Miller

### Your Email

daniel.miller@agile6.com

### Your Role and Team

Software Engineer - MYVA Auth Exp Team

### Product Manager (PM) name and email

Alexandria Parker - Alexandria.parker@agile6.com

### Product Owner (PO) name and email

Anastasia Jakabcin - ana@adhocteam.us

### Access Type Requesting

developers

### Public SSH Key

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDL0J4ZjPAIjDy4ndzt5+rzooFO2+BS87HOVryWYL5yZ1nUQnTVDm6eoq12FiMF7aOq9UioH+UPBViOfHTXdQ1RPIAgsTFgSQ7qo/xjh+Mok46/c1KY0+iI4B4ck5m/GzEQFFRGG6xU7HlwhT2IbnKit1s7meJhrFlZqJ0J0bi1AwEpn4wVnCbjlWnAq1mEJ++DJpd2sSUKHPAcGfcAEtIjuOWbi4/+VyLeVK3K52EGnYhXJX7/1kOR954Tg0XLjwv0aa92+rKsWYv2QZZrgEjl2/3faJGusjuOTlTDxjNpCxnVlclugzevx9TwZOBKyCi+1oy9ThDiDEJ/uCtPS5i3QJnrwNvqi5TusZqpzTUXeGpIOIcyICyuQcGuob5JSeIoeEIZsk+fk65KewTpe3PJavQztsXTsd7EjlYysPzvgqID2X954OP+G/kAXqVAZW8e/zI372OSrvZjVdJSFn0mYOcQsoHqdavMaD0TjeJ4PwWWdSyUWolBUtHVnk2MrYc= Daniel.Miller@Daniel-Miller.local

### E-QIP Transmittal Confirmation
<img width="895" alt="Screenshot 2023-12-08 at 4 07 56 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/54035677/49558baa-0d29-41f4-8655-7d9dccda7244">

### User must exist in a roster before SOCKS access can be granted

- [ ] Search for user on the VFS Team Roster: https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665
- [ ] Or search for user on the Platform Team Roster: https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the Platform orientation process https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [ ] Comment in this issue saying which roster the user is listed in.

**Comment by ph-One** [comment]
@DanielMiller-agile6 You should get a follow-up email [to the one shown] that shows EQIP Transmittal

**Comment by DanielMiller-agile6** [comment]
thanks Kyle.  I got a follow up email that said that could be used in place of a transmittal, but wasn't sure If I did this right :) Here's the follow up on that email:
![Screenshot 2023-12-11 at 8 46 40 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/54035677/4ff60a82-3321-4fd8-8a44-73a98d975728)

I also found my transmittal notice from 2020 when I helped out on a VA project before, but wasn't sure if this was the referenced "previous investigation" from the first email.  I've been working on a CMS contract before starting here, so I might be able to find those notices as well if needed.
![Screenshot 2023-12-11 at 8 54 07 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/54035677/86911d49-1242-46e4-9fb7-f5303bab4b40)
